src/components/taskListItem.tsx

```diff
@@ -0,0 +1,189 @@
+//react imports
+import {View, Text, StyleSheet} from 'react-native';
+import React, {useEffect, useState} from 'react';
+
+//local imports
+import themeProvider from '../providers/themeProvider';
+import getStoredTasks from '../utils/currentTasks';
+import {taskType, tasksType, task} from './types';
+import checkDate from '../utils/checkDate';
+import useIsDarkMode from '../utils/checkDarkMode';
+import convertCents from '../utils/convertCents';
+
+interface taskProps {
+  item: number;
+}
+
+export default function Task({item}: taskProps): JSX.Element {
+  // const {title, deadline, stakes} = tasksArray[Number(item)];
+  const primaryStyle = {
+    backgroundColor: useIsDarkMode()
```

Comment:
Extract to `const isDarkMode = useIsDarkMode()` to avoid multiple calls each render?

Reply:
optimized